integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/smooks/SmooksMediatorConfigFromLocalEntryTestCase.java

```diff
@@ -18,134 +18,101 @@
 package org.wso2.carbon.esb.mediator.test.smooks;
```

Comment:
Need to update the licence year.

Reply:
This is an existing class therefore we should not change the year.

Reply:
If you have made a change on the file you have to change the year. 
The year should the the year of the last edit.

Reply:
AFAIK the purpose of including the year is to show which year the code originated :)